src/main/java/xyz/nucleoid/extras/NucleoidExtras.java

```diff
@@ -62,7 +62,7 @@ public static Identifier identifier(String path) {
 
     @Nullable
     public static String getChatMessageContent(Text message) {
```

Comment:
This shouldn't be needed anymore- text content is always just the message itself and not the surrounding decorations now 🙂